packages/components/psammead-media-player/README.md

```diff
@@ -23,11 +23,13 @@ This component to be used at any point on the page, specifically when a media pl
 | Argument  | Type                | Required | Default | Example         |
 |-----------|---------------------|----------|---------|-----------------|
 | `src` | string | Yes   | - | `http://foobar.com/embeddable_endpoint` |
+| `srcset` | string | Yes   | - | `http://foobar.com/images/ic/512xn/p01k6mtv.jpg` |
```

Comment:
Does this need to be updated to `placeholderSrcset`?
Also, can I suggest we use an example more inline with srcset?